crates/ibc/src/context/token_transfer.rs

```diff
@@ -141,6 +141,32 @@ where
             .store_withdraw(token, added_withdraw)
             .map_err(TokenTransferError::from)
     }
+
+    fn store_ibc_denom(
+        &self,
+        owner: &Address,
+        coin: &PrefixedCoin,
+    ) -> Result<(), TokenTransferError> {
+        if coin.denom.trace_path.is_empty() {
```

Comment:
Why would we call the function in this case, though? If we must do it like this, we should call the function `maybe_store_ibc_denom` or something, since it sometimes doesn't store anything at all.